common/thorhelper/roxiehelper.cpp

```diff
@@ -1491,38 +1499,34 @@ bool CSafeSocket::readBlock(StringBuffer &ret, unsigned timeout, HttpHelper *pHt
             {
                 *payload = 0;
                 payload += 4;
-                char *str;
 
                 pHttpHelper->parseHTTPRequestLine(header);
+                const char *headers = strstr(header, "\r\n");
```

Comment:
@ghalliday Clients are well behaved.  If I did check for this case I would return an error to the client saying the request was invalid.